test/test_images/probe_helper/probe/handlers/ping.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2020 Google LLC
```

Comment:
2021

Reply:
Good point! Should this perhaps be covered by an automated check?

Reply:
Oh... it already is covered by the automated boilerplate check, but the boilerplate still says 2020: https://github.com/google/knative-gcp/blob/master/hack/boilerplate/boilerplate.go.txt

I will try to update it.